filebeat/input/kafka/config.go

```diff
@@ -50,6 +50,7 @@ type kafkaInputConfig struct {
 	TLS            *tlscommon.Config `config:"ssl"`
 	Username       string            `config:"username"`
 	Password       string            `config:"password"`
+	AzureLogs      string            `config:"azure_logs"`
```

Comment:
This field should be documented and validated -- consider following the pattern used for enums like `kafka.Version`, which is validated by its `Unpack` function (see `libbeat/common/kafka/version.go`) and can thereafter be used as an enum constant rather than a string match.

Reply:
@faec , I have replaced the AzureLogs with YieldEventsFromField property. This property value, if entered will have the json object name that will contain multiple events.
Ex for azure:
{ "records": [ {event1}, {event2}]}
The YieldEventsFromField property value will be "records".
I have also removed any check for the azure type logs and added some validation inside the actual pipeline.
Let me know if you think this property should be handled the same way as "kafka.Version".